jlm/llvm/opt/RvsdgTreePrinter.hpp

```diff
@@ -34,8 +41,29 @@ class RvsdgTreePrinter final : public optimization
   class Configuration final
   {
   public:
-    explicit Configuration(const util::filepath & outputDirectory)
-        : OutputDirectory_(std::move(outputDirectory))
+    enum class Annotation
+    {
+      /**
+       * Must always be the first enum value. Used for iteration.
+       */
+      FirstEnumValue,
+
+      /**
+       * Annotate regions and structural nodes with the number of RVSDG nodes.
+       */
+      NumRvsdgNodes,
+
+      /**
+       * Must always be the last enum value. Used for iteration.
+       */
+      LastEnumValue
+    };
+
+    Configuration(
```

Comment:
Documentation?

Reply:
It is the constructor of the Configuration class. What did you have in mind writing there as documentation? The only thing that pops up in my mind: "Creates an instance of the Configuration class". However, this applies for any constructor of a class and does not add any value. Do you want the parameters/attributes in the class documented?

Reply:
My bad, shouldn't have commented on this one.